### PR TITLE
 Cluster Mirror Base

### PR DESCRIPTION
**Pull request by sjug**
Initial CM functionality + README

Original CM PR https://github.com/openshift/origin/pull/19303 was closed.

### CODE REVIEW

**Comment by jeremyeder** [comment]
looks like this has my feedback from the old PR already folded in.

**Comment by jeremyeder** [comment]
lgtm/

File: test/extended/cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+reasonably approximate fidelity) a control-plane workload. The idea is to make
+sure our scale testing efforts are as realistic as possible.
+
+Use-cases include:
+
+ * Replicating OpenShift Online clusters into lab environments for scale testing and R&D.
+ * Reproducing customer environments in support situations.
+
+Note: This tool does not inspect persistent storage, it does not look at the
+content of secrets and it does not export user data in any way. It takes a
```
[the remark this answers is not in the document]
Comment (hongkailiu) [reply test/extended/cluster/README.md]: Understood. Thanks for the reply.

**Pull request merged**